Adding glossary term for the Assisted Installer and incorrect usage (#276)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

*See also*: xref:business-rule[business rule], xref:business-process[business process], xref:decision-table[decision table], xref:data-model[data model], xref:dsl[DSL]

[discrete]
[[assisted-installer]]
==== image:images/yes.png[yes] Assisted Installer (noun)

> *Description*: The Assisted Installer provides software-as-a-service functionality for user-friendly cluster installations by using the Red Hat Hybrid Cloud Console. 

Reviewer: ```suggestion
*Description*: In Red Hat OpenShift, the Assisted Installer provides Software-as-a-Service functionality for user-friendly cluster installations by using the Red Hat Hybrid Cloud Console. 
```

Reviewer: Two minor tweaks for consistency with other entries:

- Prefix with "In Red Hat OpenShift," for consistency with the **installer-provisioned infrastructure** and **user-provisioned architecture** entries.
- Use initcap for _Software_ and _Service_ for consistency with the **SaaS** entry.

Author: Implemented these fixes, thanks again!

> *Description*: In Red Hat OpenShift, the Assisted Installer provides Software-as-a-Service functionality for user-friendly cluster installations by using the Red Hat Hybrid Cloud Console. 

Reviewer: Two more small tweaks:

```suggestion
*Description*: In Red Hat OpenShift, the Assisted Installer provides Software-as-a-Service functionality for cluster installations that are hosted on the Red Hat Hybrid Cloud Console. 
```

Author: Thanks, updated

*Use it*: yes

*Incorrect forms*: AI, assisted installer

> 

Reviewer 2: Can you add the `*See also*:` line down here? We keep it even if it's empty
